app/models/middleware_server.rb

```diff
@@ -42,11 +43,11 @@ def in_domain?
 
   # Returns whether a server is immutable or not.
   #
-  # By default, we define all servers as being mutable, so that power
-  # operations are allowed, and it is the provider responsability to
+  # By default, we define all servers as being immutable, so that power
+  # operations are not allowed, and it is the provider responsability to
   # reimplement this method if necessary.
   def immutable?
-    properties['Immutable'] == 'true'
+    true
```

Comment:
@pilhuhn while I completely understand that on Hawkular, on manageiq I think this should not be the case, because having it being mutable *enables* more operations, and capabilities should only be enabled if the provider "provides" an impl for them. That way we do not run into a case where power operations are enabled for a provider that does not implement them.

What do you think?